### PR TITLE
Update community reports

### PR DESCRIPTION
**Pull request by PierreRambaud**
- Add a better logging system to retrieve requested urls and data
- Use requests-cache to store http cache data
- Use requests to continue to parse requests url in case there is a Link in the headers (When pagination is over 100)
- Make PrestaShop/PrestaShop branches configurable in a config files
- Write docstring as rtd style (https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]
Need a second review

File: README.md
Comment (jolelievre) [inline README.md]: Maybe it would be easier to have a dedicated option `--week`

File: core-weekly.py
Comment (jolelievre) [inline core-weekly.py]: Maybe `elif args.graph` ? Or we assume that graph is the default mode then it should be in the help message

File: core_weekly/github.py
Comment (jolelievre) [inline core_weekly/github.py:101]: I don't understand this sleep? Do you wait for the response to be saved in the cache?

File: core_weekly/github.py
Comment (jolelievre) [inline core_weekly/github.py]: Seems like you call the next links *only* when the response was not cached? Shouldn't it be done on each requests no matter the cache?

File: coreteam.py
Comment (jolelievre) [inline coreteam.py:55]: We don't need to add the other prestashop modules?

File: core-weekly.py
Comment (PierreRambaud) [reply core-weekly.py]: Yes I see that, Thinking to change it after because the graph fucntion doesn't do anything for now

File: coreteam.py
Comment (PierreRambaud) [reply coreteam.py:55]: When we will see it's needed.

File: core_weekly/github.py
Comment (PierreRambaud) [reply core_weekly/github.py:101]: It's because of GitHub API rate limit, which is logically to 30 requests per minute per Ip.

File: core_weekly/github.py
Comment (PierreRambaud) [reply core_weekly/github.py]: My bad, wrong indentation :D

File: README.md
Comment (jolelievre) [inline README.md]: You can add the new --week argument here

File: README.md
Comment (jolelievre) [inline README.md]:
```suggestion
 $ python ./core-weekly.py --week 11 > my-core-weekly.md
```

File: README.md
Comment (jolelievre) [inline README.md]:
```suggestion
 $ python ./core-weekly.py --year 2018 --week 11 > my-core-weekly.md
```

File: core-weekly.py
Comment (jolelievre) [inline core-weekly.py]: --week is only the number of week, not a date range any more

File: core-weekly.py
Comment (jolelievre) [inline core-weekly.py:32]: Should there be a default `else` statement to inform that the command will do nothing? Or display the help?

File: README.md
Comment (jolelievre) [inline README.md:18]: missing [--week]

**Review by jolelievre — COMMENTED** [review]
Some small changes left and it's ok for me

File: core-weekly.py
Comment (PierreRambaud) [reply core-weekly.py:32]: Not really needed

**Pull request merged**
